### PR TITLE
Add an example for LightGBM Tuner

### PR DESCRIPTION
**Pull request by jinnaiyuu**
This PR adds a simple example to show the usage of LightGBM Tuner (#846).
Following lightgbm_simple.py, catboost_simple.py, and xgboost_simple.py, the breast cancer dataset is used.

Let me know if there is any way to improve this example.

### CODE REVIEW

File: examples/lightgbm_tuner_simple.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    $ python lightgbm_tuner_simple.py
+
+"""
+
+import numpy as np
+import sklearn.datasets
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+
+import lightgbm as lgb_original
```
Comment (smly) [inline examples/lightgbm_tuner_simple.py]: In this example, `import lightgbm` can be omitted.
`optuna.integraiton.lightgbm.Dataset` can be used instead of `lightgbm.Dataset`.

```diff
diff --git a/examples/lightgbm_tuner_simple.py b/examples/lightgbm_tuner_simple.py                                                                                                                                               
index 5d325e19..1bd78e1b 100644
--- a/examples/lightgbm_tuner_simple.py
+++ b/examples/lightgbm_tuner_simple.py
@@ -14,15 +14,14 @@ import sklearn.datasets
 from sklearn.metrics import accuracy_score
 from sklearn.model_selection import train_test_split
 
-import lightgbm as lgb_original
 import optuna.integration.lightgbm as lgb
 
 
 if __name__ == '__main__':
     data, target = sklearn.datasets.load_breast_cancer(return_X_y=True)
     train_x, val_x, train_y, val_y = train_test_split(data, target, test_size=0.25)
-    dtrain = lgb_original.Dataset(train_x, label=train_y)
-    dval = lgb_original.Dataset(val_x, label=val_y)
+    dtrain = lgb.Dataset(train_x, label=train_y)
+    dval = lgb.Dataset(val_x, label=val_y)
 
     params = {
         'objective': 'binary',

Comment (jinnaiyuu) [reply examples/lightgbm_tuner_simple.py]: Good catch!
I have updated the code as you suggested.

File: examples/lightgbm_tuner_simple.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+"""
+Optuna example that optimizes a classifier configuration for cancer dataset using LightGBM tuner.
+
+In this example, we optimize the validation accuracy of cancer detection.
+We optimize both the choice of booster model and their hyperparameters.
```
Comment (sile) [inline examples/lightgbm_tuner_simple.py]: AFAIK, LightGBM Tuner doesn't optimize booster type, so this line seems somewhat misleading. Besides, I think that it's no problem to remove this line completely. What do you think of it?

File: examples/lightgbm_tuner_simple.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+"""
+Optuna example that optimizes a classifier configuration for cancer dataset using LightGBM tuner.
+
+In this example, we optimize the validation accuracy of cancer detection.
```
Comment (sile) [inline examples/lightgbm_tuner_simple.py]: Suggested a minor change to align to the script body:
```suggestion
In this example, we optimize the validation log loss of cancer detection.
```

File: examples/lightgbm_tuner_simple.py (hunk 2 above)
Comment (jinnaiyuu) [reply examples/lightgbm_tuner_simple.py]: Yea, you're right :) I'll remove this sentence.

File: examples/lightgbm_tuner_simple.py (hunk 3 above)
Comment (jinnaiyuu) [reply examples/lightgbm_tuner_simple.py]: Fixed!
